src/Flash.js

```diff
@@ -1,31 +1,35 @@
-import React from 'react'
 import PropTypes from 'prop-types'
-import classnames from 'classnames'
-import sass from 'sass.macro'
-import {injectGlobal} from 'emotion'
 import styled from 'styled-components'
-import {COMMON} from './constants'
+import {COMMON, get} from './constants'
 import theme from './theme'
 
-injectGlobal(sass`
-  @import "primer-alerts/index.scss";
-`)
-
 const schemeMap = {
-  green: 'success',
-  red: 'error',
-  yellow: 'warn'
+  green: {color: 'colors.green.8', bg: 'colors.green.1'},
```

Comment:
Might be better to just use class names for these styles instead, like I did in TextInput. Curious what you think is cleaner/more of a best practice @mxstbr?

Reply:
Ah smart! I didn't even think about the fact that adding a class might implicitly add it to the public API, good catch! For that alone I think it's a good idea to shy away from using classes!